images/ingress-nginx-controller/configs/latest.apko.yaml

```diff
@@ -0,0 +1,196 @@
+contents:
+  packages:
+    - curl
+    - curl-dev
+    - bash
+    - apk-tools
+    - busybox
+    - git
+    - build-base
+    - ca-certificates-bundle
+    - ca-certificates
+    - openssl
+    - openssl-dev
+    - ingress-nginx-controller
+    - dumb-init
+    - geoip-dev
+    - pcre
+    - zlib
+    - libxml2
+    - tzdata
+    - opentracing-cpp
+    - msgpack
+    - mimalloc=1.7.6-r2
```

Comment:
does this need to be pinned to 1.7.6-r2?  I wonder if instead it can be > 1.0 or something so rebuilds will continue to receive minor and patch releases?